src/MovieList/MovieAsset/MovieAsset.jsx

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styles from './MovieAsset.scss';
+
+function MovieAsset(props) {
+  const showDetails = () => {
+    props.showDetails(props);
```

Comment:
Destructuring for props :). And if we pass asset as one prop to the component we will need to pass `props.asset` to `showDetails`